tutor/acceptance/helpers/git.js

```diff
@@ -0,0 +1,50 @@
+const gitP = require('simple-git/promise');
+const path = require('path');
+const { imagesDir } = require('./images-dir');
+const chalk = require('chalk');
+
+const repoURL = 'https://github.com/openstax/tutor-images.git';
```

Comment:
The images are in a separate repo? One downside of this is that QA or devs will not be able to see which screenshots changed during a PR review (which I think would be helpful).

If the concern is repo size, would git-lfs help? I'm not sure if those changes would render in the PR diff though.

Reply:
Yeah, image size was the concern.  I'll investigate git-lfs, maybe it'll work for us